ddtrace/appsec/iast/taint_sinks/_base.py

```diff
@@ -18,6 +16,7 @@
 from ddtrace.appsec.iast.reporter import Source
 from ddtrace.appsec.iast.reporter import Vulnerability
 from ddtrace.internal import core
+from ddtrace.internal.compat import iteritems
```

Comment:
This refactor resolves an import error raised when module unloading and asm are both enabled